src/components/Footer.jsx

```diff
@@ -1,3 +1,36 @@
-const Footer = () => {};
+import { HiLink } from "react-icons/hi";
+import { IoMdMail } from "react-icons/io";
+import { PiTiktokLogo } from "react-icons/pi";
+import { FaDiscord } from "react-icons/fa";
+import { AiFillInstagram } from "react-icons/ai";
+import Link from "next/link";
 
+const Footer = () => {
+  return (
+    <div className="flex w-screen p-2 mt-[10%] bg-biscuits-peach justify-between text-biscuits-purple-200 font-omc text-xl">
```

Comment:
items should be centered vertically. you can fix it using items alignment.

https://tailwindcss.com/docs/align-items

Reply:
i added the same padding used in the nav bar to align the footer too and it looks aligned lmk if this is fine!
![image](https://github.com/acm-ucr/h4b-website/assets/136354400/41736447-907e-4fb8-888f-f4bdc639dd43)

Reply:
it still looks a bit misaligned. use items-center instead of using padding.

i would also try to increase size of the org name and icons a bit

Reply:
![image](https://github.com/acm-ucr/h4b-website/assets/91030482/a2659237-cb11-4b1c-b88d-7ed1e914dd03)

Reply:
ok i added items-center but it still looks a little off compared to the nav bar 
![image](https://github.com/acm-ucr/h4b-website/assets/136354400/8620ee38-6ae5-4f09-8048-6215e4c50caa)